apps/codelab/src/app/codelabs/extra/feedback-page/feedback-message-table/feedback-message-table.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, Input, ViewChild, Output, EventEmitter } from '@angular/core';
+import { MatMenuTrigger } from '@angular/material';
+
+
+@Component({
+  selector: 'codelab-feedback-message-table',
+  templateUrl: './feedback-message-table.html',
+  styleUrls: ['./feedback-message-table.css'],
```

Comment:
Consider using OnPush here?
https://netbasal.com/a-comprehensive-guide-to-angular-onpush-change-detection-strategy-5bac493074a4